index.css

```diff
@@ -91,7 +92,9 @@ body {
 }
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/219133533-d0d70ff9-d9d8-4de9-8395-584903f46067.png)

- [ ] Your navbar and social links are not perfectly aligned to right :fearful: It would be nice if you would fix it as suggested by the previous reviewer :+1: